src/integrationTest/resources/application.yml

```diff
@@ -5,6 +5,8 @@ spring:
     platform: h2
     username: ${POSTGRES_USERNAME:dbuserprofile}
     password: ${POSTGRES_PASSWORD:dbuserprofile}
+  jpa:
+    show-sql: true
   main:
     allow-bean-definition-overriding: true
```

Comment:
u can remove this from production

Reply:
this is in the integration aplication.yml file it wont harm in the production

Reply:
removed it